src/Sarif.Multitool.Library/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -246,6 +246,18 @@ internal static string JsonPointerToJavaScript(string pointerString)
             return sb.ToString();
         }
 
+        internal static bool UriIsWellFormedUriString(string uriString, UriKind uriKind)
```

Comment:
>UriIsWellFormedUriString [](http://example.com/codeflow?start=29&length=24)

create a unit test that checks for the previous behavior and the new behavior.

This would guarantee that even if we remove the E2E tests, we would still be able to validate it.